### ISSUE TITLE
Can't have 2 MPASLI compset names

### ISSUE THREAD

**Issue by rljacob** [E3SM-Project/E3SM#2417]
CIME technically doesn't allow for 2 different compset names to point the same config_component.xml.

In this case, MPAS land ice uses two different compset names:  MALI and MALISIA in
```
2000_SATM_DLND%SCPL_SICE_SOCN_SROF_MALI_SWAV
RCP8_SATM_SLND_SICE_SOCN_SROF_MALISIA_SWAV

```
But the components/mpas-albany-landice/cime_config/config_component.xml has GLC_COMP set to mali.

CIME does not enforce this (see https://github.com/ESMCI/cime/issues/2459) and the only error is that short term archiving is quietly not done for MALISIA.

The solution is to define the SIA case using `%` modifiers.
`RCP8_SATM_SLND_SICE_SOCN_SROF_MALI%SIA_SWAV`

**Comment by rljacob** [comment]
Tagging @jonbob  and @matthewhoffman .  @jgfouca already has done this and will included it in https://github.com/E3SM-Project/E3SM/pull/2414.  New tests in that PR found the problem.

**Comment by jgfouca** [comment]
Fixed by #2414
